site/quickstart.md

```diff
@@ -162,7 +162,9 @@ domain namespace (`sample-domain1-ns`) and the `domainHomeImageBase` (`oracle/we
 
 * Setting `weblogicCredentialsSecretName` to the name of the secret containing the WebLogic credentials.
   By convention, the secret will be named`domainUID-weblogic-credentials` (where `domainUID` is replaced with the
-  actual `domainUID` value).
+  actual `domainUID` value). 
+
+* Leaving the `image` empty unless you need to tag the new image that the script builds to a different name.
```

Comment:
Leaving the `image` empty -> Leave the `image` empty

Reply:
I was trying to follow the existing pattern here. Shall we also change the two bullets above this one too?

Copying the sample -> Copy the sample
Setting weblogicCredentialsSecretName -> Set weblogicCredentialsSecretName

Reply:
Oh sorry, I got that wrong, didn't look at the other bulleted items. You were correct to follow the existing pattern of using the gerund form of the verb.